frontend/src/components/FacilityForm/FacilityForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import Facility from "../Facility/Facility";
+import { View, Pressable, Text } from "react-native";
+import { styles } from "./FacilityFormStyles";
+import Button from "../Button/Button";
+import { Ionicons } from "@expo/vector-icons";
+import PopUpModal from "../Modal/PopUpModal";
+
+const FacilityForm = () => {
+  const [facilityList, setFacilityList] = useState([]);
+  const [isModalVisible, setModalVisible] = useState(false);
+
+  const exampleList = [
```

Comment:
A really small thing but can this be moved to the top outside the component function so that it only runs once since its value will never change.